docs/TESTS.md

```diff
@@ -1,7 +1,37 @@
 # Tests
 
-When working offline, first make sure [pyret-npm] is installed and available. Then execute tests by passing `pyret` the location of the test file relative to your current working directory.
-`$ pyret relative/path/to/example-test.arr`
+When working locally (i.e. not within the Exercism web editor), first make sure [pyret-npm] is installed and accessible. Then execute the tests by passing `pyret` the location of the test file relative to your current working directory like
+`$ pyret relative/path/to/example-test.arr`.
+
+On this track, Pyret check blocks are wrapped in functions to delay detection and execution. Those functions are stored as a TestRun value with a field for the function and another field for whether it should be run. When working locally, this second field should be set to `true` instead of `false` if you want to run a specific test. On the Exercism website, all TestRun values are run regardless of that second field's contents so a solution passing locally may not pass on the website if all the local tessts haven't been enabled.
```

Comment:
```suggestion
On this track, Pyret check blocks are wrapped in functions to delay detection and execution. Those functions are stored as a `TestRun` value with a field for the function and another field for whether it should be run. When working locally, this second field should be set to `true` instead of `false` if you want to run a specific test. On the Exercism website, all TestRun values are run regardless of that second field's contents so a solution passing locally may not pass on the website if all the local tessts haven't been enabled.
```